src/lib/mongoBackend/connectionOperations.cpp

```diff
@@ -0,0 +1,710 @@
+/*
+*
+* Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Fermín Galán
+*/
+
+#include "mongoBackend/MongoGlobal.h"
+#include "mongoBackend/connectionOperations.h"
+#include "logMsg/traceLevels.h"
+
+#include "mongo/client/dbclient.h"
+
+using namespace mongo;
+
+// FIXME P5: this macro is also in other parts of the code (grep for 'FT(x)'). It should
```

Comment:
Also (I just did a search), ngsi/ContextAttribute.cpp defines its own FT(x) - probably my fault :-)

Feel free to join all these (and if not, we should probably create an issue about it)